collector/client.go

```diff
@@ -231,3 +265,76 @@ func decodeServiceTimeStrings(line string) (types.Counter, error) {
 
 	return types.Counter{}, errors.New("service times - could not parse line: " + line)
 }
+
+func decodeInfoStrings(line string) (types.Counter, error) {
+	if strings.HasSuffix(line, ":\n") { // A header line isn't a metric
+		return types.Counter{}, nil
+	}
+
+	if equal := strings.Index(line, ":"); equal >= 0 {
+		if key := strings.TrimSpace(line[:equal]); len(key) > 0 {
+			value := ""
+			if len(line) > equal {
+				value = strings.TrimSpace(line[equal+1:])
+			}
+			key = strings.Replace(key, " ", "_", -1)
+			key = strings.Replace(key, "(", "", -1)
+			key = strings.Replace(key, ")", "", -1)
+			key = strings.Replace(key, ",", "", -1)
+			key = strings.Replace(key, "/", "", -1)
+
+			// metrics with string save as label
+			if key == "Squid_Object_Cache" || key == "Build_Info" || key == "Service_Name" {
+				if key == "Squid_Object_Cache" {
+					key = key + "_Version"
+					if slices := strings.Split(value, " "); len(slices) > 0 {
+						value = slices[1]
+					}
+				}
+				var infoVarLabel types.VarLabel
+				infoVarLabel.Key = key
+				infoVarLabel.Value = value
+
+				var infoCounter types.Counter
+				infoCounter.Key = key
+				infoCounter.VarLabels = append(infoCounter.VarLabels, infoVarLabel)
+				return infoCounter, nil
+			} else if key == "Start_Time" || key == "Current_Time" {
+				return types.Counter{}, nil
+			}
+
+			// Remove additional formating string
+			if slices := strings.Split(value, " "); len(slices) > 0 {
+				if slices[0] == "5min:" {
```

Comment:
can we also collect 60m counters?

Reply:
yes, I push the changes in order to collect 5m and 60m counters